### ISSUE TITLE
fix: FAST foundation breadcrumb item template missing `.control` container

### ISSUE THREAD

**Issue by hawkticehurst** [microsoft/fast#4856]
# 🐛 Bug Report

I'm currently using the FAST Foundation breadcrumb item for a project and realized that the following CSS style––which was referenced from the FAST Component [breadcrumb item styles](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-components/src/breadcrumb-item/breadcrumb-item.styles.ts#L86-L89)––is not being applied/rendered:

```
.control:not([href]) {      
    cursor: default;    
}
```

Instead of a default cursor being rendered, a text cursor is rendered when hovering over a breadcrumb item which does not have an `href` attribute.

After some investigation, I discovered that the FAST Foundation breadcrumb item [template](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-foundation/src/breadcrumb-item/breadcrumb-item.template.ts#L19-L26) does not include a container element with the class `control` in the instance where an `href` attribute is not included.

## 💻 Repro or Code Sample

This issue can actually be seen in FAST Component [demonstration site](https://explore.fast.design/components/fast-breadcrumb) where the breadcrumb item at the end with no `href` renders `cursor: text` on hover instead of `cursor: default`.

## 🤔 Expected Behavior

That the CSS styles mentioned above would be rendered.

## 😯 Current Behavior

The CSS styles mentioned above are not rendered.

## 💁 Possible Solution

My assumption is that wrapping the `slot` element in [this template](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-foundation/src/breadcrumb-item/breadcrumb-item.template.ts#L19-L26) with `<span class="control">` will solve this issue.

If that is the case I'd be willing to contribute to that change.

## 🔦 Context

In addition to getting the `cursor: default` style to render, I would also love to be able to add focus border styling to the breadcrumb item content when the `href` attribute is not applied.

## 🌍 Your Environment

* OS & Device: MacOS
* Browser: Google Chrome
* Version: 1.22.0

**Comment by chrisdholt** [comment]
Thanks for the issue @hawkticehurst! I think you are correct in that the problem here is that in the example given, `.control` is part of the `Anchor` template being interpolated but *not* part of the text slot. However, I wonder if we could approach this without a breaking change. Rather than looking for href on the control, what if we were looking for a host without an href? Technically, the slot that this is being passed into could also be used for slotted content which is customized - I think the solution we're looking for here is the default behavior which would be no href on the host, thus a text only node. Thoughts on that?

**Comment by hawkticehurst** [comment]
Ooh yeah touché. Working from a place where we look for a host without an `href` seems like a really solid direction!

In fact, I just went down a rabbit hole exploring this direction and seemed to fix the problem I was having!

I started by writing the following ruleset:

```css
:host(:not([href]) {      
    cursor: default;    
}
```

This worked well except that the styles were being applied to the entire host instead of just the content (i.e. text node) within the default `slot`.

With that in mind I added `slot:not([name])` to the above selector like so:

```css
:host(:not([href]) slot:not([name]) {      
    cursor: default;    
}
```

This does actually solve the problem of rendering `cursor: default` correctly (yay). However, it still doesn't address the other use case I mentioned of being able to add a focus border when no `href` is added. I discovered this was because `slot` seems to be an inline element by default, so by adding a `display: block` declaration I finally addressed all the issues I mentioned above. 

```css
:host(:not([href]) slot:not([name]) {    
    cursor: default; 
    display: block;
    ... now box model related declarations can be applied ...
}
```

Since the FAST Breadcrumb Item also has [this issue](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-components/src/breadcrumb-item/breadcrumb-item.styles.ts#L86-L93) with rendering the styles of default `slot` content I'd still be happy to open a PR that updates the selectors to be `:host(:not([href]) slot:not([name])` if you'd like?

I'm also not sure if the last bit of CSS changing the default `slot` to be a block element is something you on the FAST team are looking for in your Breadcrumb Item, but at least this solves the issues I was having so I'll add it in on my end.

Let me know what you think.
